doc/source/user/basics.copies.rst

```diff
@@ -81,9 +81,9 @@ For example::
     >>> y.base is None
     True
 
-    Here, ``y`` is a copy, as signified by the :attr:`base <.ndarray.base>`
-    attribute. We can also confirm this by assigning new values to ``x[[1, 2]]``
-    which in turn will not affect ``y`` at all::
+Here, ``y`` is a copy, as signified by the :attr:`base <.ndarray.base>`
```

Comment:
Maybe give a little more context to the significance of the `base` attribute? Something like
```suggestion
Here, ``y`` is a copy, since the :attr:`base <.ndarray.base>` is no longer ``x``
```

Reply:
Oh, I'm just fixing the formatting. Changing the content is beyond the scope of this PR.